<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QGroupBox" name="CPUBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>741</width>
          <height>121</height>
         </rect>
        </property>
        <property name="title">
         <string>CPU</string>
        </property>
        <widget class="QProgressBar" name="cpuProgressBar">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>60</y>
           <width>118</width>
           <height>23</height>
          </rect>
         </property>
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="NetworkBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>190</y>
          <width>741</width>
          <height>131</height>
         </rect>
        </property>
        <property name="title">
         <string>Network Usage</string>
        </property>
        <widget class="QLabel" name="networkRxLabel">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>40</y>
           <width>631</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <widget class="QLabel" name="networkTxLabel">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>80</y>
           <width>681</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="MemoryBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>360</y>
          <width>741</width>
          <height>131</height>
         </rect>
        </property>
        <property name="title">
         <string>Memory Usage</string>
        </property>
        <widget class="QProgressBar" name="memoryProgressBar">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>60</y>
           <width>118</width>
           <height>23</height>
          </rect>
         </property>
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="DiskBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>530</y>
          <width>741</width>
          <height>141</height>
         </rect>
        </property>
        <property name="title">
         <string>Disk Usage</string>
        </property>
        <widget class="QLabel" name="ioReadLabel">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>50</y>
           <width>671</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <widget class="QLabel" name="ioWriteLabel">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>90</y>
           <width>681</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
